example_project_bzl_4/third_party/ncurses/defs.bzl

```diff
@@ -0,0 +1,17 @@
+load("@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl", "nixpkgs_package")
+load("//third_party/nix:defs.bzl", "NIX_REPOSITORIES")
+
+def ncurses():
```

Comment:
I'm thinking we might need a small re-think of this to simplify. I've tried putting the 2 packages under a filegroup so we have a single @ncurses//:ncurses target => might put a transition perhaps to join them together. Idea is that we can just do @ncurses instead of @ncurses.headers and @ncurses.lib as a dependency (and obviously have @ncurses//:headers when needed.